markdown/blog/websocket-part2.md

```diff
@@ -165,7 +165,7 @@ servers:
       The API client must request an authentication "token" via the following REST API endpoint "GetWebSocketsToken" to connect to WebSockets Private endpoints. For more details, read https://support.kraken.com/hc/en-us/articles/360034437672-How-to-retrieve-a-WebSocket-authentication-token-Example-code-in-Python-3
 
       The resulting token must be provided in the "token" field of any new private WebSocket feed subscription: 
-      ```
+      
```

Comment:
Why this Codeblock is removed?

Reply:
Please Go to the production url https://www.asyncapi.com/blog/websocket-part2
You will see the following error message in the blog content. 
> Application error: a client-side exception has occurred (see the browser console for more information).

After Fixing the i18n configuration, the most part of the site started to render on the server. So any inconsistency in the markdown can be caught during the build. 

When we kept the codeblocks, the following error was occuring during build
 
![image](https://github.com/user-attachments/assets/3ae387dc-f01c-4fea-916b-49f7d1c7dd0a)

Reply:
But we show that code as part of codeblock. Don't we have any solution for it?

Reply:
Sambhav has pushed solution. PTAL. Thanks.